src/actions/editor.ts

```diff
@@ -53,7 +53,7 @@ export type Action =
   | SetDebugPaneSize
   | ShowLogs
   | SetCompiledVegaPaneSize
-  | SetCurrentTheme
+  | SetCurrentConfig
```

Comment:
Why not just SetConfig and SetThemeName?

Reply:
Will do!